Automattic/jetpack
# User Management: Tracking events

Fixes https://github.com/Automattic/wp-calypso/issues/87608

## Proposed changes:

Add tracking events for User management, `/wp-admin/users.php`, SSO changes.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync to Atomic site
* Navigate to `/wp-admins/users.php`.
* Trigger the SSO user actions (Revoke Invite, Send Invite) and add a new user.
* Navigate to the Tracks Live View (MC: `/tracks/live/`) , filter by your username, and verify the events are being tracked correctly.
